guacamole/src/main/webapp/app/rest/types/Error.js

```diff
@@ -43,6 +43,15 @@ angular.module('rest').factory('Error', [function defineError() {
         this.message = template.message;
```

Comment:
It's not used directly by the JavaScript anymore, but it continues to exist as a means of providing a human readable message alongside the machine-readable translation key. It can be helpful during debugging.